pandas/tests/test_graphics.py

```diff
@@ -1329,10 +1329,6 @@ def test_plot(self):
                                      subplots=True, use_index=False)
         self._check_axes_shape(axes, axes_num=4, layout=(4, 1))
 
-        df = DataFrame({'x': [1, 2], 'y': [3, 4]})
```

Comment:
It raises ``TypeError`` properly on Mac with mpl 1.5.2. You can use ``compat.is_platform_linux()`` and mpl >= 1.5.2 to skip the test.

Also, pls add original issue number for reference.